lib/version_utils.pm

```diff
@@ -476,7 +476,7 @@ sub is_using_system_role {
       && (install_this_version() || install_to_other_at_least('12-SP2'))
       || (is_sles4sap() && main_common::is_updates_test_repo())
       || is_sle('=15')
-      || (is_sle('>15') && (check_var('SCC_REGISTER', 'installation') || get_var('ADDONS') || get_var('ADDONURL')))
+      || (is_sle('>15') && (check_var('SCC_REGISTER', 'installation') || get_var('ADDONS') || get_var('ADDONURL') || get_var('BASE_VERSION')))
```

Comment:
Could you please help me to understand this change. Did it work for SLE 15 SP1 or we didn't have this scenario?
I just wonder if we should put some condition like BASE_VERSION = 15 GA, or it works in other conditions because `VERSION` variable matched `(is_sle('>15')`?

Also, could you please add description of `BASE_SYSTEM` variable to [variables.md](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/variables.md).